modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultCodegen.java

```diff
@@ -1294,6 +1295,14 @@ public String apiFilename(String templateName, String tag) {
     }
 
     public boolean shouldOverwrite(String filename) {
-        return true;
+        return !(skipOverwrite && new File(filename).exists());
```

Comment:
this change in semantics from "should overwrite" to "should write" breaks `JaxRSServerCodegen` (and doesn't correspond with the method name); addressed in https://github.com/ahgittin/swagger-codegen/commit/8fadce27d7208564a16bad98175e210c19648a00